code/game/machinery/vending.dm

```diff
@@ -962,21 +963,22 @@
 
 /obj/machinery/vending/wallmed1
 	name = "\improper NanoMed"
-	desc = "A wall-mounted version of the NanoMed."
+	desc = "A wall-mounted version of the NanoMed, stocked with the equipment necessary for stabilisation and much more."
 	product_ads = "Go save some lives!;The best stuff for your medbay.;Only the finest tools.;Natural chemicals!;This stuff saves lives.;Don't you want some?"
 	icon_state = "wallmed"
 	icon_deny = "wallmed-deny"
 	icon_vend = "wallmed-vend"
 	base_type = /obj/machinery/vending/wallmed1
 	density = FALSE //It is wall-mounted, and thus, not dense. --Superxpdude
 	products = list(
-		/obj/item/stack/medical/bruise_pack = 3,
-		/obj/item/stack/medical/ointment = 3,
-		/obj/item/reagent_containers/pill/paracetamol = 4,
-		/obj/item/storage/med_pouch/trauma,
-		/obj/item/storage/med_pouch/burn,
-		/obj/item/storage/med_pouch/oxyloss,
-		/obj/item/storage/med_pouch/toxin
+		/obj/item/stack/medical/bruise_pack = 2,
+		/obj/item/stack/medical/advanced/bruise_pack = 4,
+		/obj/item/stack/medical/ointment = 6,
+		/obj/item/reagent_containers/pill/paracetamol = 8,
+		/obj/item/storage/med_pouch/trauma = 2,
+		/obj/item/storage/med_pouch/burn = 2,
+		/obj/item/storage/med_pouch/oxyloss = 2,
+		/obj/item/storage/med_pouch/toxin = 2,
```

Comment:
Explain all the balance changes here.

Reply:
one is not enough- incase reinjection might be needed, two is good enough for people not to abuse this vending machine.

Reply:
im a medical main, someone snags all these med_pouches first thing.

Reply:
Bad

Reply:
how is this bad.